common/nginx/etc/nginx.conf

```diff
@@ -12,6 +12,12 @@ http {
 
                 error_page 502 @502;
 
+                location /friendbot {
```

Comment:
The friendbot in the file above where friendbot is moved to `common`, and there is this change that exposes friendbot on all networks. Why is that? Friendbot is only useful on a standalone network, not on futurenet or testnet, etc.

For accessing friendbot locally when connected to other networks we need to fix #57.

Reply:
I tried to avoid maintaining permutations of nginx conf per each network, it led to duplication in each, rather one complete conf here which sets up rules for all possible services, in each network permutation, will only launch via supervisor conf the backing services relevant, so any rules here that don't have a backing service running, just return 503

Reply:
I missed your reply here. We actually need a different section for friendbot for each network anyway for #57, so running separate nginx is preferable. We can merge this as is and follow up.

Reply:
But also, the `standalone/friendbot/etc/friendbot.cfg → common/friendbot/etc/friendbot.cfg` change still seems unnecessary. We only ever run friendbot for standalone, so there shouldn't be any need to run it on another network configuration so it should stay in standalone. We can merge this as is and clean this up after.